packages/demo/react-sdk-demo-app/package.json

```diff
@@ -14,15 +14,15 @@
   "dependencies": {
     "@phantom/browser-sdk": "workspace:*",
     "@phantom/react-sdk": "workspace:*",
+    "@vitejs/plugin-react": "^4.4.1",
     "react": "^19.1.0",
-    "react-dom": "^19.1.0"
+    "react-dom": "^19.1.0",
+    "vite": "^6.3.5"
```

Comment:
It doesn't seem correct to mark build tool as a dependency here, it should be kept as a dev dependency